src/modules/previewpane/SvgPreviewHandler/SvgPreviewControl.cs

```diff
@@ -183,10 +189,22 @@ private void AddWebViewControl(string svgData)
                         await _browser.CoreWebView2.AddScriptToExecuteOnDocumentCreatedAsync("window.addEventListener('contextmenu', window => {window.preventDefault();});");
                         _browser.CoreWebView2.SetVirtualHostNameToFolderMapping(VirtualHostName, AssemblyDirectory, CoreWebView2HostResourceAccessKind.Allow);
                         _browser.CoreWebView2.Settings.AreDefaultScriptDialogsEnabled = false;
-                        _browser.NavigateToString(svgData);
+
+                        // WebView2.NavigateToString() limitation. See docs.
+                        if (svgData.Length > 2 * 1024 * 1024)
```

Comment:
Should it be > or >= ? Did you test actual limitation?
Maybe add https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.navigatetostring?view=webview2-dotnet-1.0.864.35#remarks to the comments here for reference.
(I think there's a pattern for this already in `.github/actions/spell-check/patterns.txt` )